tools/Microsoft.TemplateEngine.Authoring.TemplateVerifier/TemplateVerifierOptions.cs

```diff
@@ -126,6 +126,11 @@ public class TemplateVerifierOptions : IOptions<TemplateVerifierOptions>
         /// </summary>
         public bool DoNotPrependCallerMethodNameToScenarioName { get; init; }
 
+        /// <summary>
+        /// <see langword="true" />, if the output directory has to be empty before the test execuion.
+        /// </summary>
+        public bool VerifyOutputDirectoryEmptiness { get; init; }
```

Comment:
```suggestion
        public bool EnsureEmptyOutputDirectory { get; init; } = true;
```
I would suggest def behavior as true. To allow non-empty directory, false should be explicitly specified.

Reply:
Agreed here - functional change should be opt-in, otherwise it's a breaking change